cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,76 @@
 /// <reference types='cypress' />
 
+const { firstName } = require('faker/lib/name');
+const { generateUser } = require('../support/generate');
```

Comment:
I don't see such file in your commit
```suggestion
const { generateUser } = require('../support/generate');
```